src/store/configureStore.js

```diff
@@ -0,0 +1,16 @@
+import { applyMiddleware, createStore, combineReducers, compose } from 'redux'
+import thunkMiddleware from "redux-thunk";
+import {legislators} from "../reducers/legislators";
+
+const reducer = combineReducers({
+  legislators,
+});
+
+const createStoreWithMiddleware = compose(
+  applyMiddleware(thunkMiddleware),
```

Comment:
by default redux is all synchronous, redux-thunk allows asynchronous actions in redux.